resources/scripts/components/dashboard/forms/SetupTwoFactorModal.tsx

```diff
@@ -12,13 +12,24 @@ import Button from '@/components/elements/Button';
 import asModal from '@/hoc/asModal';
 import ModalContext from '@/context/ModalContext';
 import QRCode from 'qrcode.react';
+import Spinner from '@/components/elements/Spinner';
+import CopyOnClick from '@/components/elements/CopyOnClick';
 
 interface Values {
     code: string;
 }
 
 const SetupTwoFactorModal = () => {
-    const [ token, setToken ] = useState('');
+    const [ tokenUrl, setTokenUrl ] = useState('');
```

Comment:
Just return the token from the API rather than having this code to parse it out of the generated URL.

Reply:
I did not want to break BC by changing what the API returns.